otx/algorithms/action/adapters/mmaction/models/backbones/movinet.py

```diff
@@ -0,0 +1,1102 @@
+"""
+Code inspired by:
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv2.html
+https://pytorch.org/vision/stable/_modules/torchvision/models/mobilenetv3.html
+"""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from collections import OrderedDict
+import torch
+from torch.nn.modules.utils import _triple, _pair
+import torch.nn.functional as F
+from typing import Any, Callable, Optional, Tuple, Union
+from einops import rearrange
+from torch import nn, Tensor
+from yacs.config import CfgNode
+from mmaction.models.builder import BACKBONES
+
+
+class Hardsigmoid(nn.Module):
```

Comment:
How about moving general pytorch functions (`Hardsigmoid`, `Swish`) to the `common/adapters/torch/`? In this case, I think `common/adapters/torch/activation_functions/` is the proper location. @harimkang , @goodsong81, @jaegukhyun

Reply:
Replaced with torch.nn methods, but if you have good idea, please share on this thread :)